iaso/api/bulk_create_users.py

```diff
@@ -366,6 +382,23 @@ def module_permissions(current_account):
         # Get and return all permissions linked to the modules
         return account_module_permissions(account_modules)
 
+    @staticmethod
+    def validate_phone_number(phone_number):
```

Comment:
There is no unit test for this function.

You should at least test that `validate_phone_number()` works as expected in the CSV import tests.